src2/t.js

```diff
@@ -0,0 +1,35 @@
+while (foo1) {
+}
+
+while (!!foo1_v1) {
```

Comment:
Redundant double negation.

```suggestion
while (foo1_v1) {
```